contracts/ProductNft/ProductNft.sol

```diff
@@ -0,0 +1,170 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
```

Comment:
If there's no specific reason you want this version, let's go with the latest solidity version (`0.8.15`) now.